applications/GeoMechanicsApplication/geo_mechanics_application.h

```diff
@@ -601,6 +602,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) KratosGeoMechanicsApplication : publ
     const GeoThermalDispersionLaw mGeoThermalDispersion2DLaw{ConstitutiveLaw::SizeType(2)};
     const GeoThermalDispersionLaw mGeoThermalDispersion3DLaw{ConstitutiveLaw::SizeType(3)};
 
+    const GeoThermalFilterLaw mGeoThermalFilter2DLaw{ConstitutiveLaw::SizeType(1)};
+    const GeoThermalFilterLaw mGeoThermalFilter3DLaw{ConstitutiveLaw::SizeType(1)};
```

Comment:
It is not clear for me to use **ConstitutiveLaw**::SizeType. Can it be just SizeType?